tests/next_tests/integration_tests/feature_tests/ffront_tests/test_execution.py

```diff
@@ -93,6 +93,19 @@ def testee(a: cases.VField) -> cases.EField:
     )
 
 
+@pytest.mark.uses_unstructured_shift
+def test_unstructured_shift(unstructured_case):
```

Comment:
```suggestion
def test_horizontal_only_with_3d_mesh(unstructured_case_3d):
  # test field operator operating only on horizontal fields while using an offset provider
  # including a vertical dimension.
```
See comment above. Additionally is there a reason this a neighbor reduction and not the simpler `a(E2V[0])`?

Reply:
Is this triggered only from unstructured? Do you need a shift?